packages/mds-ingest-service/service/validators.ts

```diff
@@ -62,9 +65,10 @@ export const { validate: validateDeviceDomainModel, $schema: DeviceSchema } = Sc
         default: []
       },
       modality: { type: 'string', enum: MODALITIES, default: 'micromobility' },
-      year: { type: 'integer' },
-      mfgr: { type: 'string' },
-      model: { type: 'string' }
+      // ⬇⬇⬇ NULLABLE/OPTIONAL PROPERTIES ⬇⬇⬇
+      year: nullableInteger,
```

Comment:
```suggestion
      year: { ... nullableInteger, minimum: 1980, maximum: new Date().getFullYear() + 2 },
```
This would be a future proof version of what used to be there.

Reply:
Yeah, I was thinking about that, and @janedotx had a similar suggestion.. honestly, I don't know if just `+2` years is going to be sufficient forever, and I reeeeeally hate picking arbitrary numbers like that. Worst-case-scenario, this could be caught after-the-fact when analyzing technical compliance, as opposed to at ingestion. While a stupidly high year wouldn't make much sense, it's not going to _break_ anything, so I dunno if that validation belongs at the ingestion layer. 

I'm happy to re-implement the year thresholding (in a more lenient fashion than it was previously), but I just want to make sure we're 100% sure we should do it, and that this is the place to do it :)

Cc. @marie-x feedback very solicited!